src/model/enums/mod.rs

```diff
@@ -0,0 +1,18 @@
+//! All Enums for Rspotify
+pub mod country;
+pub mod misc;
+pub mod types;
+
+pub use country::Country;
+
+pub use misc::DisallowKey;
```

Comment:
Can these be grouped up with `use thing::{a,b,c}`?

Reply:
Sure!